### ISSUE TITLE
github issue/subreddit competeting in posix and browsers

### ISSUE THREAD

**Issue by SRS-WRKS** [martin12333/browserPOSIX-discussion#10]
Greetings,

GDSR :
This seems like a very nice project. I was just made aware of it and i had to respond , because i am chained to my keyboard. The shackles are wearing themselves in, thank you for asking.
 
POSIX supports everything as a file architecture ( plan9,etc). while web needs http links to be "browser friendly".

Unix tools support a variety of protocols while porting posix tools to work in Javascript+web asm would result in "maintainable code" optimized for environment or architecture (JS doesn't care about architecture, but please bear with me).

Would this be the analogous starting point of when terminal support for programs were removed, because of it this project succeeds, we would be living in the world where everything requires a link and not all protocols work with links, just like place on earth requires a GPS coordinates. It is possible to have a nice vacation if the tour itinerary promises enough good places with good reviews right ??

**Comment by SRS-WRKS** [comment]
TLDR: There is a unix haters handbook, is there a HTTP hates handbook yet ??
